app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def category_taxons
```

Comment:
These changes are already on `main` so shouldn't be here, please rebase this branch on `main` and push it up again